migrations/versions/a1b2c3d4e5f6_initial_schema.py

```
@@ -8,6 +8,7 @@
 
 from collections.abc import Sequence
 from typing import Union
+from os import getenv
```

Flake8 (7.2.0)

[error] 'os.getenv' imported but unused

(F401)